ap/plotting/plotter.py

```diff
@@ -0,0 +1,181 @@
+# coding: utf-8
+
+"""
+Generalized plotting functions to create plots from hist histograms
+"""
+
+from ap.util import maybe_import
+
+np = maybe_import("numpy")
+hist = maybe_import("hist")
+plt = maybe_import("matplotlib.pyplot")
+mplhep = maybe_import("mplhep")
+
+
+def draw_error_stairs(ax, h, kwargs={}):
```

Comment:
Could you rather use `kwargs=None` as a default and the set it to an empty dict when None?

```python
def draw_error_stairs(ax, h, kwargs=None):
    if kwargs is None:
        kwargs = {}
```

Mutable default arguments can become [nasty](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments) 👀

Reply:
Yes, I'll add the changes to the next commit